index.md

```diff
@@ -2,15 +2,16 @@
 layout: page
 title: Flutter
 ---
-Flutter is a new way to build high-performance, cross-platform mobile apps.
-Flutter is optimized for today's, and tomorrow's, mobile devices. We are focused
-on low-latency input and high frame rates on Android and iOS.
+
+Flutter is a new project to help developers
+build high-performance, high-fidelity,
+cross-platform mobile apps.
```

Comment:
This text exists in several places.  Do we want to update it everywhere?

https://github.com/flutter/engine/blob/master/README.md
https://github.com/flutter/flutter/blob/master/README.md
https://github.com/flutter/engine/blob/master/sky/packages/sky/README.md